README.md

```diff
@@ -1,2 +1,59 @@
 # Salesforce-Postman
 Salesforce API Postman Collection
+
+# How to install it
+
+Go to the [release section](https://github.com/scolladon/Salesforce-Postman/releases) to download the salesforce-apis.postman_export.json from the release you want.
```

Comment:
Add direct download link for the latest release: "Download the latest version here or browse through previous versions"